packages/moonstone/styles/colors.less

```diff
@@ -105,8 +105,8 @@
 
 // FormCheckbox
 // ---------------------------------------
-@moon-formcheckbox-bg-color: @moon-button-bg-color;
-@moon-formcheckbox-text-color: @moon-button-text-color;
+@moon-formcheckbox-bg-color: fade(@moon-white, 22%);
```

Comment:
needs documentation to verify

Reply:
This was _"funny"_ to me too, –that it's not just a round number–, but IIRC, 22% opacity was the level that visually matched the same color tone of GUI's existing graphic design.